src/Physics/InitialField.cpp

```diff
@@ -186,6 +186,12 @@ void seissol::physics::SnellsLaw::evaluate(double time,
 #endif
 }
 
+seissol::physics::Ocean::Ocean(int mode, double gravitationalAcceleration)
```

Comment:
Could you maybe add a reference to this test? Google does not give too much information, if I search for it ;-)

Reply:
Added. But the reference isn't really that helpful, because it's an AGU abstract - the results will be published by L. Abrahams et al. soonish

Reply:
Actually, I've added a reference to a very similar 2D scenario. Will update once other paper is published (at least as preprint)

Reply:
:+1: